sphinx/ext/imgconverter.py

```diff
@@ -52,9 +53,10 @@ def is_available(self):
             stdout, stderr = p.stdout.read(), p.stderr.read()
             p.wait()
         if p.returncode != 0:
+            encoding = sys.getdefaultencoding()
             logger.warning(__('convert exited with error:\n'
                               '[stderr]\n%s\n[stdout]\n%s'),
-                             (stderr, stdout))
+                           stderr.decode(encoding), stdout.decode(encoding))
```

Comment:
I worried about this `decode` is needed or not. So I'm waiting for @usaturn 's feedback.

Reply:
With current implementation, decode error is raised.
```
Encoding error:
'utf-8' codec can't decode byte 0x96 in position 0: invalid start byte
The full traceback has been saved in C:\Users\gohki\AppData\Local\Temp\sphinx-err-r5gsafxm.log, if you want to report the issue to the developers.
```

Note: `sys.getdefaultencoding()` returns `'utf-8'`. It seems `locale.getpreferredencoding()` is better in this case.